### PR TITLE
Features/bug list 2

### PR DESCRIPTION
**Pull request by shivangi-gupta123**
Spectator bugs list

### CODE REVIEW

File: src/Components/spectatorDashboard/rooms/RoomForm/index.tsx [hunk 1]
```diff
@@ -26,7 +26,9 @@ const CreateRoomForm = (props) => {
       validationSchema,
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       onSubmit: async (values, { resetForm }) => {
-        const dateTimeString = new Date(`${values.date} ${values.time}`);
+
+        const dateTimeString = new Date(`${values.date}T${values.time}`);
+        console.log("dateTimeString----->", dateTimeString)
```
Comment (Ashok-tgc) [inline src/Components/spectatorDashboard/rooms/RoomForm/index.tsx]: remove console.log

File: src/Components/spectatorDashboard/rooms/RoomForm/index.tsx [hunk 2]
```diff
@@ -26,7 +26,9 @@ const CreateRoomForm = (props) => {
       validationSchema,
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (Ashok-tgc) [inline src/Components/spectatorDashboard/rooms/RoomForm/index.tsx]: remove this line

File: src/app/spectatorDashboard/Room/page.tsx [hunk 3]
```diff
@@ -62,6 +62,7 @@ const Room = () => {
       const spectatorResponse = await sendRequest('room/user-rooms', {
         method: 'GET',
       });
+      console.log("spectatorResponse----->", spectatorResponse)
```
Comment (Ashok-tgc) [inline src/app/spectatorDashboard/Room/page.tsx]: remove console.log

File: src/Components/spectatorDashboard/rooms/RoomForm/index.tsx (hunk 1 above)
Comment (shivangi-gupta123) [reply src/Components/spectatorDashboard/rooms/RoomForm/index.tsx]: done

File: src/Components/spectatorDashboard/rooms/RoomForm/index.tsx (hunk 2 above)
Comment (shivangi-gupta123) [reply src/Components/spectatorDashboard/rooms/RoomForm/index.tsx]: done

File: src/app/spectatorDashboard/Room/page.tsx (hunk 3 above)
Comment (shivangi-gupta123) [reply src/app/spectatorDashboard/Room/page.tsx]: done

**Pull request merged**